package on2012_11.on2012_11_19_CROC_MBTU_Elimination_Round.E___Mishap_in_Club;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2012_11/on2012_11_19_CROC_MBTU_Elimination_Round/E___Mishap_in_Club/E - Mishap in Club.task"))
			Assert.fail();
	}
}
